v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/schema/ISchemaMapper.java

```diff
@@ -94,7 +94,8 @@ List<String> getSpannerColumns(String namespace, String spannerTable)
   String getSyntheticPrimaryKeyColName(String namespace, String spannerTableName);
 
   /**
-   * Returns true if the Spanner column exists at the source.
+   * Returns true if the Spanner column exists at the source. For identity mappers, since there is
```

Comment:
nit: this is specific to the identity mapper, we probably should not put it here.